Answers.md

```diff
@@ -1,3 +1,15 @@
 ### Your responses to the short answer questions should be laid out here using markdown.
 
 For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+1. A virtual machine is an OS running on top of an OS in a virtual environment. Windows running on a Mac whilst having OSX as the main OS, is just one example. Docker creates a copy of the dev environment
+
+2. The port numbers for Localhost
+
+3. So you can spin up more machines when a specific machine needs more memory, CPU, or storage. Also allows you to shut down inactive machines so there are no wasted resources.
+
+4. kubectl scale --replicas=2
+
+5. Horizontal means adding more machines to use more resources. Vertically means adding more CPU or RAM to resources.
+
+6. Limitations on availability in regards to resources. 
```

Comment:
Additionally, the free tier containers sleep after 30 minutes of inactivity. When a new request comes in to a sleeping container, the container must wake up, meaning it can't respond to the request as quickly. Thus, the free tier containers are less performant overall.